sonar-javascript-plugin/src/main/java/org/sonar/plugins/javascript/JavaScriptPlugin.java

```diff
@@ -73,8 +73,8 @@ public class JavaScriptPlugin implements Plugin {
   public static final String LCOV_REPORT_PATHS_DEFAULT_VALUE = "";
 
   public static final String ENVIRONMENTS = "sonar.javascript.environments";
-  public static final String ENVIRONMENTS_DEFAULT_VALUE = "amd, applescript, atomtest, browser, commonjs, couch, embertest, flow, greasemonkey, jasmine, jest, jquery, " +
-    "meteor, mocha, mongo, nashorn, node, phantomjs, prototypejs, protractor, qunit, rhino, serviceworker, shared-node-browser, shelljs, webextensions, worker, wsh, yui";
+  public static final String ENVIRONMENTS_DEFAULT_VALUE = "amd,applescript,atomtest,browser,commonjs,couch,embertest,flow,greasemonkey,jasmine,jest,jquery," +
+    "meteor,mocha,mongo,nashorn,node,phantomjs,prototypejs,protractor,qunit,rhino,serviceworker,shared-node-browser,shelljs,webextensions,worker,wsh,yui";
```

Comment:
```suggestion
  public static final String ENVIRONMENTS_DEFAULT_VALUE = String.join(",", "amd","applescript","atomtest","browser","commonjs","couch","embertest","flow","greasemonkey","jasmine","jest","jquery",      "meteor","mocha","mongo","nashorn","node","phantomjs","prototypejs","protractor","qunit","rhino","serviceworker","shared-node-browser","shelljs","webextensions","worker","wsh","yui");
```

I think this would be less error prone in the long-term

Reply:
Much better indeed, thanks!